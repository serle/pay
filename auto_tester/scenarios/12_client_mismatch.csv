type,client,tx,amount
deposit,1,1,100.0
deposit,2,2,200.0
dispute,2,1,
resolve,2,1,
chargeback,2,1,
