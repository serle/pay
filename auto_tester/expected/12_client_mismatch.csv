client,available,held,total,locked
1,100.0000,0.0000,100.0000,false
2,200.0000,0.0000,200.0000,false
